lib/pymedphys/_pinnacle/pinnacle.py

```diff
@@ -70,8 +70,10 @@ def __init__(self, path, logger=None):
 
         # Show a warning to ensure that this module is used with caution
         warnings.warn(
-            "The Pinnacle Export Module is intended for research purposes only, "
-            "use with caution.",
+            "The Pinnacle Export Module is intended for research purposes only. Parts "
+            "of the DICOM conversion are hard-coded and completely ignore your data. "
+            "This has proven useful for research applications but is not appropriate "
+            "for clinical use.",
```

Comment:
Thanks @pchlap, that significantly reduces my stress levels. Thank you :slightly_smiling_face:.